tests/Entity/AttributeTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace YotiTest\Entity;
+
+use Yoti\Entity\Attribute;
+use YotiTest\TestCase;
+
+class AttributeTest extends TestCase
+{
+    const ATTR_NAME = 'full_name';
+
+    const ATTR_VALUE = 'Test FullName';
+
+    const ATTR_SOURCES = [
+        'PASSPORT'
+    ];
+
+    const ATTR_VERIFIERS = [
+        'YOTI_ADMIN'
+    ];
+
+    /**
+     * @var Attribute
+     */
+    public $dummyAttribute;
+
+    public function setup()
+    {
+        $this->dummyAttribute = new Attribute(
+            self::ATTR_NAME,
+            self::ATTR_VALUE,
+            self::ATTR_SOURCES,
+            self::ATTR_VERIFIERS
+        );
+    }
+
+    public function testAttributeName()
+    {
+        $this->assertEquals(self::ATTR_NAME, $this->dummyAttribute->getName());
+    }
+
+    public function testAttributeValue()
+    {
+        $this->assertEquals(self::ATTR_VALUE, $this->dummyAttribute->getValue());
+    }
+
+    public function testAttributeSources()
+    {
+        $this->assertEquals(
+            json_encode(self::ATTR_SOURCES),
```

Comment:
Is it necessary to encode twice?

Reply:
We are comparing 2 arrays, so the best way is to compare there JSON strings.